docs/content/manual/deployment/4480-config-sftp-dynamically.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Config sftp dynamically
+---
+
+* Related issues: [#4480](https://github.com/harvester/harvester/issues/4480) [ENHANCEMENT] config sftp dynamically
+
+
+## Category: 
+* Deployment
+
+## Verification Steps
+1. Use ipex-example to provision Harvester
```

Comment:
typo, should be `ipxe`.

Reply:
Thanks for the check.
Correct the repo in line 12.